quickwit/quickwit-actors/src/actor_handle.rs

```diff
@@ -158,13 +158,13 @@ impl<A: Actor> ActorHandle<A> {
     /// as the finalize logic may behave differently depending on the exit status.
     pub async fn kill(self) -> (ActorExitStatus, A::ObservableState) {
         self.actor_context.kill_switch().kill();
-        let _ = self
```

Comment:
Ok, I think you're right in removing this send kill message. It seems that it was not necessary to send this kill message as in the actor loop we always check if the kill switch is activated. Plus in the in actor handle, the implementation can check if the kill switch too. So in the end, activating the kill switch is always more powerful.